docs/articles/documentation/recipes/debug-in-webstorm.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: docs
+title: Debug in WebStorm
+permalink: /documentation/recipes/debug-in-webstorm.html
+---
+# Debug in WebStorm
+
+This topic describes how to debug your TestCafe tests in [WebStorm](https://www.jetbrains.com/webstorm/) with a Node.js debugger.
```

Comment:
your TestCafe tests - I think we can just write 'TestCafe tests' because this applies to all.